crates/configuration/src/configuration.rs

```diff
@@ -60,14 +60,29 @@ pub async fn parse_configuration(
     environment: impl Environment,
 ) -> Result<Configuration, Error> {
     let configuration_file = configuration_dir.as_ref().join(CONFIGURATION_FILENAME);
-    let configuration_file_contents = fs::read_to_string(&configuration_file).await?;
-    let configuration: version3::RawConfiguration =
+
+    let configuration_file_contents =
+        fs::read_to_string(&configuration_file)
+            .await
+            .map_err(|err| {
+                Error::IoErrorButStringified(format!("{}: {}", &configuration_file.display(), err))
```

Comment:
I just want rust to tell me which file it couldn't read, so I added these everywhere. Sorry.